lms/djangoapps/django_comment_client/utils.py

```diff
@@ -147,7 +147,7 @@ def get_discussion_id_map_entry(xblock):
         xblock.discussion_id,
         {
             "location": xblock.location,
-            "title": xblock.discussion_category.split("/")[-1].strip() + " / " + xblock.discussion_target
+            "title": xblock.discussion_category.split("/")[-1].strip() + (" / " + xblock.discussion_target if xblock.discussion_target else "")
```

Comment:
Just curious why we need to first split by `/` and then add `/` again. Do `discussion_category` already have `discussion_target` in it and we are just reversing the order?
Have you checked what `xblock.discussion_category.split("/")[-2].strip()` (if it exists)  holds?